test/unit/org/apache/cassandra/utils/CassandraGenerators.java

```diff
@@ -482,7 +482,12 @@ public TableParamsBuilder withFastPathStrategy()
                         int size = SourceDSL.integers().between(1, Integer.MAX_VALUE).generate(rnd);
                         map.put(ParameterizedFastPathStrategy.SIZE, Integer.toString(size));
                         Set<String> names = new HashSet<>();
-                        Gen<String> nameGen = SourceDSL.strings().allPossible().ofLengthBetween(1, 10).assuming(s -> !s.trim().isEmpty());
+                        Gen<String> nameGen = SourceDSL.strings().allPossible().ofLengthBetween(1, 10)
+                                                       // If : is in the name then the parser will fail; we have validation to disalow this
+                                                       .map(s -> s.replace(":", "_"))
+                                                       // Names are used for DCs and those are seperated by ,
+                                                       .map(s -> s.replace(",", "_"))
```

Comment:
these are the real bug fixes.

`:` caused `dc:weight` parser to fail (we expect 2, but had 3)
`,` caused issues as we had a dc `a,b`

These are generator bugs and not allowed from CQL layer... so switched to `_` to avoid shrinking the size